nevergrad/optimization/test_utils.py

```diff
@@ -108,8 +108,8 @@ def test_pruning() -> None:
 @pytest.mark.parametrize(  # type: ignore
     "nw,dimension,expected_min,expected_max",
     [
-        (12, 8, 100, 1000),
-        (24, 8, 168, 1680),
+        (12, 8, 52, 156),
+        (24, 8, 52, 156),
```

Comment:
you should add cases that matter here.
In the initial examples, there were 2 different cases (hence 2 different expected max), but in your examples it the same case twice.